configs/native.yaml

```diff
@@ -13,6 +13,7 @@ data:
       download_url: 'http://robotics.ethz.ch/~asl-datasets/ijrr_euroc_mav_dataset/vicon_room1/V1_02_medium/V1_02_medium.zip'
 demo_data: demo_data/
 loader:
+  kimera_path: .cache/paths/https%c%s%sgithub.com%sILLIXR%sKimera-VIO/
```

Comment:
Don't harcode. What if someone wants to run a local copy of Kimera and not the git repo one? This shouldn't be a part of the config. This should be determined and set by the runner.

Reply:
It is fine that this is hardcoded and there is a future issue to make this less ugly. This is the best we can do right now.

Reply:
People will most certainly make changes to Kimera-VIO during the hackathon (instrumentation, etc.) and this would become problematic then. We can ask them to update this path manually but I don't understand what the blocker is in doing this from inside the runner. The runner always knows the full path because it builds the plugins.

Reply:
They wouldn't have to change this during the hackathon.

Reply:
If they do we can show them how. The changes to the runner are bigger than just a couple of days of work.

Reply:
@mhuzai Let's get this one in and push another PR correcting it--if we have time.